github/users.go

```diff
@@ -134,6 +134,55 @@ func (s *UsersService) Edit(ctx context.Context, user *User) (*User, *Response,
 	return uResp, resp, nil
 }
 
+// HovercardOptions specifies optional parameters to the UsersService.GetHovercard
+// method.
+type HovercardOptions struct {
+	// SubjectType specifies the additional information to be received about the hovercard.
+	// Possible values are: organization, repository, issue, pull_request. (Required when using subject_id.)
+	SubjectType string `url:"subject_type"`
+
+	// SubjectID specifies the ID for the SubjectType. (Required when using subject_type.)
+	SubjectID string `url:"subject_id"`
+}
+
+// UserContext represents the contextual information about user.
+type UserContext struct {
+	Message *string `json:"message,omitempty"`
+	Octicon *string `json:"octicon,omitempty"`
+}
+
+// Hovercard holds the parsed response from GetHovercard.
```

Comment:
It'd be nice if we could better describe what this struct is, rather than saying it's "what GetHovercard returns". Perhaps something like this?

```
// Hovercard represents hovercard information about a user.
```

It's not great, so you can go either way here.